tests/test_main.py

```diff
@@ -7,7 +7,7 @@
 from db.models import Actor, Genre, Movie, MovieSession, CinemaHall
 from services.movie import get_movies, get_movie_by_id, create_movie
 from services.cinema_hall import get_cinema_halls, create_cinema_hall
-from services.movie_session import (
+from services.movie_sessions import (
```

Comment:
Shouldn't this module be named `movie_session`?

Reply:
Oh, you're right. Can't remember exactly why I've renamed it.
Fixed 👍